src/components/FeatureViews/AdminPortal/customTestPanel.js

```diff
@@ -0,0 +1,48 @@
+import React, {useEffect, useMemo, useState} from "react";
+import {connect, useDispatch, useSelector} from "react-redux";
+import {bindActionCreators} from "redux";
+import * as adminActions from "../../../actions/adminActions";
+
+import ReactDOM from 'react-dom';
+
+import PanelManager from "./PanelManager";
+import Paper from "@material-ui/core/Paper";
+import TextField from "@material-ui/core/TextField";
+import Grid from "@material-ui/core/Grid";
+import {Button, Typography} from "@material-ui/core";
+
+function CustomTestPanel() {
+    const dispatch = useDispatch();
+    const api = useMemo(
+        () => ({
+            ...bindActionCreators(adminActions, dispatch),
+        }),
+        [dispatch]
+    );
+    
+    const fields = {
+
+    };
+    const fetchFunction =[
+    api.fetchMultiCourseDiscount(),
+    api.fetchPaymentMethodDiscount(),
+    api.fetchDateRangeDiscount()
```

Comment:
Does PanelManager handle this properly...? If I understand this correctly you will be displaying all types of discounts under 1 panel manager instead of having 3 panel managers display 3 types of discounts.